bitnami/kafka/values.yaml

```diff
@@ -612,6 +612,31 @@ sidecars: []
 ##
 initContainers: []
 
+networkPolicy:
+  ## @param networkPolicy.enabled Enable creation of NetworkPolicy resources.
+  ##
+  enabled: false
+  ## @param networkPolicy.allowExternal Don't require client label for connections
+  ## The Policy model to apply. When set to false, only pods with the correct
+  ## client label will have network access to Kafka port for client connections (service.port).
+  ## When true, kafka will accept connections from any source
+  ## (with the correct destination port).
+  ##
+  allowExternal: true
+  ## @param networkPolicy.explicitNamespacesSelector A Kubernetes LabelSelector to explicitly select namespaces from which traffic could be allowed
+  ## If explicitNamespacesSelector is missing or set to {}, only client Pods that are in the networkPolicy's namespace
+  ## and that match other criteria, the ones that have the good label, can reach the kafka.
+  ## But sometimes, we want the kafka to be accessible to clients from other namespaces, in this case, we can use this
+  ## LabelSelector to select these namespaces, note that the networkPolicy's namespace should also be explicitly added.
+  ##
+  ## Example:
+  ## explicitNamespacesSelector:
+  ##   matchLabels:
+  ##     role: frontend
+  ##   matchExpressions:
+  ##    - {key: role, operator: In, values: [frontend]}
+  ##
+  explicitNamespacesSelector: {}
```

Comment:
I'd include all these parameters under the section below "Exposure parameters"

Reply:
Use the [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool then to update the README.md table to reflect the changes.

Reply:
This is not done, right?

Reply:
hi, i have just move networkPolicy paramerters under Exposure section and i have re generate readme.